app/src/sm_at_socket.c

```diff
@@ -258,6 +272,103 @@ static void delegate_poll_event(struct sm_socket *s, int events)
 	}
 }
 
+static void send_cb_fn(struct k_work *work)
+{
+	for (int i = 0; i < SM_MAX_SOCKET_COUNT; i++) {
+		if (socks[i].fd == INVALID_SOCKET) {
+			continue;
+		}
+		if (atomic_get(&socks[i].send_ntf.ready) != 0) {
+			int status = socks[i].send_ntf.status;
+			int bytes_sent = socks[i].send_ntf.bytes_sent;
+
+			atomic_clear(&socks[i].send_ntf.ready);
+			urc_send("\r\n#XSEND_NTF: %d,%d,%d\r\n", socks[i].fd, status, bytes_sent);
```

Comment:
I think I would rename this to `#XSENDNTF` since we don't have any commands or notifications with underscore yet.